code/modules/antagonists/changeling/changeling.dm

```diff
@@ -585,3 +585,7 @@
 
 /datum/antagonist/changeling/xenobio/antag_listing_name()
 	return ..() + "(Xenobio)"
+
+/datum/antagonist/changeling/check_mastery()
+	if(purchasedpowers == list())
```

Comment:
Couldn't you proceed like you normally would by buying powers the whole round and then just respec at the round end to get this?